solution/ui/regulations/eregs-vite/src/components/resources/ResourcesResults.vue

```diff
@@ -94,56 +96,7 @@ export default {
         },
     },
 
-    beforeCreate() {},
-
-    created() {},
-
-    beforeMount() {},
-
-    mounted() {},
-
-    beforeUpdate() {},
-
-    updated() {},
-
-    beforeDestroy() {},
-
-    destroyed() {},
-
-    data() {
-        return {
-            dataProp: "value",
-        };
-    },
-
-    filters: {
-        locationLabel(value) {
```

Comment:
Template filters are no longer supported in Vue 3, so in anticipation of the eventual upgrade from Vue 2 to Vue 3, I'm moving some filters out into a global filters file and just importing them into components and using them as normal JS functions.

See more here: https://wildermuth.com/2020/05/11/Where-Did-Filters-Go-in-Vue-3/